alot/buffers.py

```diff
@@ -85,7 +85,7 @@ def rebuild(self):
             self.isinitialized = True
 
         lines = list()
-        displayedbuffers = filter(self.filtfun, self.ui.buffers)
```

Comment:
for those changes in this patch that replace the `map`s, OK, I kind of see your point that it may seem more natural in the way you proposed. But IMO `filter(filtfun, list)` is looks much cleaner than `[t for t in list]`..